packages/x-date-pickers/src/DesktopNextDatePicker/DesktopNextDatePicker.types.ts

```diff
@@ -24,6 +24,11 @@ export interface DesktopNextDatePickerProps<TDate>
   extends BaseNextDatePickerProps<TDate>,
     DesktopOnlyPickerProps<TDate>,
     BaseNextNonStaticPickerExternalProps {
+  /**
+   * Years rendered per row.
+   * @default 4
+   */
+  yearsPerRow?: 3 | 4;
```

Comment:
Do you really think it's worth redefining these props just to have a more correct `default`?
I wouldn't mind having the following default. 🤔 
```
`4` on desktop, `3` on mobile.
```

Reply:
If you use directly `YearCalendar` or `DateCalendar` this JSDoc would be wrong :/